### DISCUSSION TITLE
How to get logged in user with activeOrder.

### DISCUSSION THREAD

**Discussion by ivsaneesh** [vendurehq/vendure#1607] [Q&A]
I am using Vue for frontend. Logged-in using login api, but not able to find customer after login. Tried using activeCustomer and me API.
What I have done is login the customer, then it sends the user to home page. From there, i am not getting the logged-in user details and active order of that user.

**Comment by michaelbromley** [comment]
Are you using the [VueStorefront integration](https://github.com/vuestorefront-community/vendure)?

**Reply by ivsaneesh** [reply]
No, Not using VueStorefront

**Reply by michaelbromley** [reply]
ok. Sounds like a session management issue. Are you using cookies or bearer tokens to manage sessions?

**Reply by ivsaneesh** [reply]
cookies

**Reply by ivsaneesh** [reply]
It will be good to have a video tutorial of full application development from login to check out. Can you post a video on YouTube. It will be helpful to developers like us

**Reply by ivsaneesh** [reply]
What is the best way to keep the session. Is there an issue with using cookie.

**Reply by michaelbromley** [reply]
You can run into issues with cookies if the Vendure server is running on a different domain to your storefront. You need to google "SameSite cookies" and read about it. Alternatively you can use the bearer token method which requires a bit more setup of your storefront graphql client, but does not suffer from these SameSite issues. See https://www.vendure.io/docs/storefront/configuring-a-graphql-client/

**Comment by ivsaneesh** [comment]
authOptions: {
        superadminCredentials: {
            identifier: 'superadmin',
            password: 'superadmin',
        },
        tokenMethod: 'cookie',
        cookieOptions: {
          secret: process.env.COOKIE_SECRET || 'cookie-secret',
        },
    },

**Reply by ivsaneesh** [reply]
mutation {
  login(username: "sam@yopmail.com", password: "String", rememberMe: true) {
    ... on CurrentUser {
      identifier
    }
    __typename
  }
}
